src/main/java/edu/ucla/library/services/metadata/MetadataSetter.java

```diff
@@ -365,7 +365,7 @@ private void addMetadata(final String... aRow) throws FileNotFoundException, FfP
             }
         } catch (final IOException details) {
             // We want to distinguish between this IOException and one that comes from reading/writing the CSV file
-            throw new FfProbeException(details, filePath);
+            System.err.println(LOGGER.getMessage(MessageCodes.MG_106, filePath, details.getMessage()));
```

Comment:
After this change, does this method still throw `FfProbeException`?

Also, it doesn't look like it throws `FileNotFoundException` (a subclass of `IOException`), since any thrown by `getFullFilePath` would be caught by this catch block.

Reply:
Removed obsolete throws